lib/live_ex_webrtc/publisher.ex

```diff
@@ -127,6 +127,13 @@ defmodule LiveExWebRTC.Publisher do
   @type on_recording_finished :: (publisher_id :: String.t(), Recorder.end_tracks_ok_result() ->
                                     any())
 
+  @typedoc """
+  Called when recorder sends a message to the Publisher.
+
+  For exact meaning of the second argument, refer to `t:ExWebRTC.Recorder.message/0`.
+  """
+  @type on_recorder_message :: (publisher_id :: String.t(), Recorder.message() -> any())
```

Comment:
How this relates to `on_recording_finished`? Are both callbacks needed?

Reply:
Personally, I would opt for separate callbacks but that's not a big deal so we can also go for messages.

If callbacks are used, I would try not to force the user to move to the recorder docs. Instead I would add `on_recording_upload_finished` with `status` field.